packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -77,6 +77,8 @@ class SimControl {
     // 1 second.
     final List<String> args = <String>[_xcrunPath, 'instruments', '-w', deviceName, '-t', 'Blank', '-l', '1'];
     printTrace(args.join(' '));
+    // Synchronization is managed separately below.
+    // ignore: unawaited_futures
```

Comment:
but i'm not sure i agree that we should be ignoring this future. Why don't we want the Process object? If nothing else, we need to be able to report if the simulator fails to run, right? We need to report an unexpected exitCode.